bootloader/ubootenv/env_test.go

```diff
@@ -25,94 +25,113 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"strings"
 	"testing"
 
 	. "gopkg.in/check.v1"
 
 	"github.com/snapcore/snapd/bootloader/ubootenv"
+	"github.com/snapcore/snapd/testutil"
 )
 
 // Hook up check.v1 into the "go test" runner
 func Test(t *testing.T) { TestingT(t) }
 
-type uenvTestSuite struct {
+type uenvBaseSuite struct {
 	envFile string
 }
 
-var _ = Suite(&uenvTestSuite{})
+func (s *uenvBaseSuite) SetUpTest(c *C) {
+	s.envFile = filepath.Join(c.MkDir(), "uboot.env")
```

Comment:
given it's now shared, should we use a different filename, like ubootenv or something